database/database_test.go

```diff
@@ -0,0 +1,12 @@
+package database
+
+const (
+	// MongoTestUsername is the username for the test database
+	MongoTestUsername = "admin"
+
+	// MongoTestPassword is the password for the test database
+	MongoTestPassword = "aO4tV5tC1oU3oQ7u"
+
+	// MongoTestConnString is the connection string for the test database
+	MongoTestConnString = "mongodb://localhost:37017"
```

Comment:
I don't think those fields need to be exported?

Reply:
we do actually, other packages will eventually need to construct a test database as well,
currently this is not the case yet so I indeed removed this file and made them private for the time being